iconoir.com/pages/support.tsx

```diff
@@ -98,25 +82,97 @@ const Support: NextPage<SupportProps> = ({ ...headerProps }) => {
   );
 };
 
+const CenteredParagraph = styled(Text18)`
+  text-align: center;
+  margin: 120px auto;
+`;
+
+const PillarsContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  ${media.md} {
+    flex-direction: row;
+  }
+`;
+
+const Pillar = styled.div`
+  height: fit-content;
+  width: 80%;
+  border: 1px solid var(--g6);
+  background: var(--white);
+  border-radius: 12px;
+  display: flex-column;
+  text-align: center;
+  padding: 60px 10%;
+  margin-bottom: 20px;
+  ${media.md} {
+    width: 22%;
+    height: 180px;
+    padding: 60px 5%;
+    margin-bottom: 0;
+    &:hover {
+      transform: scale(1.05);
+      transition: 0.2s
```

Comment:
Moving to root so it takes effect on mouse out as well
```suggestion
    transition: 0.2s;
    &:hover {
      transform: scale(1.05);
```

Reply:
I intentionally wanted to have a faster transition on `mouse out` to have a quicker interaction with the button. But GTM, let's see how this looks, no strong point 🙌